docs/content/how-to/kubevirt/ingress-and-dns.md

```
@@ -75,95 +142,162 @@ NAME            VERSION   KUBECONFIG                       PROGRESS   AVAILABLE
 
-Now that we have the ingress exposed, next step is configure a wildcard DNS A record or CNAME that references the LoadBalancer Service's external IP.
+Configure a wildcard DNS A record or CNAME that references the LoadBalancer
+Service's external IP:
 
-1. Get the external IP.
+1. Get the external IP:
 
-  ```shell
-  export EXTERNAL_IP=$(oc -n clusters-$CLUSTER_NAME get service $CLUSTER_NAME-apps -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-  ```
+    ```shell
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)